libcxx/docs/ReleaseNotes/19.rst

```diff
@@ -46,6 +46,7 @@ Implemented Papers
 - P2869R4 - Remove Deprecated ``shared_ptr`` Atomic Access APIs from C++26
 - P2872R3 - Remove ``wstring_convert`` From C++26
 - P3142R0 - Printing Blank Lines with ``println`` (as DR against C++23)
+- P2944R3 - Comparisons for ``reference_wrapper`` (comparison operators for ``reference_wrapper`` only)
```

Comment:
Do you plan to implement the rest of the paper too?

Reply:
Yes, probably in another PR later. I won't mind if somebody else does it if they wish so.